rightteaminc/rightstyle-js
# PV-73 added Area & Pie charts

https://pollywog.atlassian.net/browse/PV-73

### Issue
``Briefly explain why what this PR addresses or why it is needed``
Needed a visual of our new lovely AnyShart library. 

### Solution
``Briefly explain the solution or approach``
Journeyed through the docs of AnyShart to identify and learn the proper ways of AnySharting.

Added 4 charts to Storybook.

### Testing Notes

``What automated tests cover the changes. If no tests were added/updated, explain why.``
make test

### QA Notes
QA Resource:

Steps to reproduce and/or verify changes

npm install

npm run dev

### Side Effects
- [ ] Does this change introduce breaking changes to apis, etc? *If so or not sure, please communicate to team for [semver](https://semver.org/) bump*
- [ ] Did working on this ticket spawn other tickets, tech debt, or cruft. If so, please list ticket numbers.

### Dev Checklist
 - [ ] There is a Jira Ticket for this changes
 - [ ] The Jira ticket is assigned to me
 - [ ] The Jira ticket is pointed
 - [ ] The Jira Ticket is on the [board](https://pollywog.atlassian.net/secure/RapidBoard.jspa?projectKey=PV) and "In Code Review"
 - [ ] I have run automated tests and manually tested my change prior to tagging others for code review
 - [ ] I have run `make build` on the last commit (if applicable)
 - [ ] All TODO comments have associated tickets

### Code Reviewer Checklist
- I have reviewed the code changes +1
- I have checked out the code and run it locally +10
- I have deployed the code and manually tested +100
- I have verified the Jira ticket associated with this PR is correct and has required fields

## Merging Checklist
- Do not not squash without checking
- Please delete the branch on merged
- Ensure Jira ticket is closed and in "Done" column on the [board](https://pollywog.atlassian.net/secure/RapidBoard.jspa?projectKey=PV)

Note: Pollywog PR template v0.1.0 - If making changes, please consider updating the other repositories as well
